extensions/src/AWSSDK.Extensions.CrtIntegration/AWSSDK.Extensions.CrtIntegration.Net35.csproj

```diff
@@ -37,7 +37,7 @@
   </Choose>
 
   <ItemGroup>
-    <PackageReference Include="AWSCRT-AUTH" Version="0.4.3" />
+    <PackageReference Include="AWSCRT-AUTH" Version="0.4.4" />
     <PackageReference Include="AWSCRT-CHECKSUMS" Version="0.4.3" />
```

Comment:
`AWSCRT-CHECKSUMS` also has a `0.4.4` version available: https://www.nuget.org/packages/AWSCRT-CHECKSUMS/0.4.4

Should it be updated as well?

Reply:
Yeah, we should update all of them to 0.4.4. CRT typically releases them together, and the actual native code is all in "AWSCRT".